controllers/v1/observationsController.js

```diff
@@ -1159,11 +1301,7 @@ module.exports = class Observations extends Abstract {
                         req.rspObj.userToken
                     );
 
-                    if(userOrganisationDetails.success && userOrganisationDetails.data) {
-                        usersKeycloakIdMap = userOrganisationDetails.data;
-                    } else {
-                        throw new Error(messageConstants.apiResponses.USER_ORGANISATION_DETAILS_NOT_FOUND);
-                    }
+                    usersKeycloakIdMap = userOrganisationDetails.data;
```

Comment:
@aman-tunerlabs - What if organisations data is not there ?

Reply:
@aks30  If some issue is there on organisations api we have a try check in shikshalokamHelper only. so it will go to catch-catch chain . So here we dont need to check.